.storybook/storyshots.test.js

```diff
@@ -0,0 +1,18 @@
+import initStoryshots from '@storybook/addon-storyshots';
+
+describe('', () => {
+  // Our localhosts have a different locale than CI. Since the snapshots
+  // contain localized dates (eg. TxReceipt), we stub the call and neutralize
+  // localization.
+  beforeAll(() => {
+    jest.spyOn(global.Date.prototype, 'toLocaleString').mockImplementation(function (args) {
+      return new Date(this.valueOf()).toISOString();
+    });
+  });
+
```

Comment:
`"fontSize": undefined`

In snapshot

Reply:
I find 83 occurrences within the snap file.
The theme is correctly imported in the decorators, and the components are able to import `FONT_SIZE`. This seems out of scope of this PR.